stylesheets/screens/cart.css

```diff
@@ -32,9 +32,6 @@
     background: var(--gold-color);
 }
 
-.cart-items{
-    flex: 0 0 calc(100vw - 400px);
-}
 
 .cart-price {
     flex: 0 0 300px;
```

Comment:
why flex basis in px and not rem

Reply:
I wanted an absolute value so I have given in px here